services/API-service/src/api/admin-area-data/dto/upload-admin-area-data.dto.ts

```diff
@@ -26,3 +37,27 @@ export class UploadAdminAreaDataDto {
   @IsNumber()
   public value: number;
 }
+
+export class UploadAdminAreaDataJsonDto {
+  @ApiProperty({ example: 'ETH' })
+  @IsNotEmpty()
+  @IsString()
+  public countryCodeISO3: string;
```

Comment:
use referenced entity instead of any string. See example,
https://github.com/rodekruis/IBF-system/blob/e572308ac1d4b6f68994305a571a99b7690fb0dc/services/API-service/src/api/admin-area/admin-area.entity.ts#L17-L22